tools/example-module/example-module.template

```diff
@@ -1,43 +1,20 @@
 /**
  ******************************************************************************
  * DO NOT MANUALLY EDIT THIS FILE. THIS FILE IS AUTOMATICALLY GENERATED.
- *
- * IMPORTANT: Due to a bug with the bazel ng_module rule, this file is now
- * checked into the repository. Please rebuild the example module by running
- * the following command: "yarn gulp build-examples-module"
- *
- * BUG: https://github.com/angular/angular/issues/30259
  ******************************************************************************/
 
-import {NgModule} from '@angular/core';
-import {MAT_FORM_FIELD_DEFAULT_OPTIONS} from '@angular/material/form-field';
-
-${exampleImports}
-
 export interface LiveExample {
   title: string;
-  component: any;
+  component: string;
```

Comment:
Which value does into this string? Would it be more accurate to name it something like `componentClassName` or `componentId`?

Reply:
Changed to `componentName`. let me know if that works for you. I kept `additionalComponents` as is because it should be clear from the context.

Reply:
Happy to change that too if you have a good idea. I couldn't think of a good name. Using something like `additionalComponentNames` is incorrect.